### PR TITLE
Update notify data to use a map instead of a JSON string

### PR DESCRIPTION
**Pull request by jefflinwood**

### CODE REVIEW

**Comment by jefflinwood** [comment]
Fix for https://github.com/TwilioDevEd/sdk-starter-java/issues/27

File: exampleVideoInvite/src/main/java/com/twilio/video/examples/videoinvite/notify/api/model/Invite.java [hunk 1]
```diff
@@ -9,4 +12,13 @@ public Invite(final String fromIdentity, final String roomName) {
         this.fromIdentity = fromIdentity;
         this.roomName = roomName;
     }
+
+    public Map<String, String> getMap() {
+        HashMap<String, String> map = new HashMap<>();
```
Comment (ktalebian) [inline exampleVideoInvite/src/main/java/com/twilio/video/examples/videoinvite/notify/api/model/Invite.java]: If it's immutable, then `ImmutableMap.of("fromIdentity", fromIdentity, "roomName", roomName);`

Comment (ktalebian) [reply exampleVideoInvite/src/main/java/com/twilio/video/examples/videoinvite/notify/api/model/Invite.java]: also make the strings `fromIdentity` and `roomName` private static const of this class and use that

Comment (jefflinwood) [reply exampleVideoInvite/src/main/java/com/twilio/video/examples/videoinvite/notify/api/model/Invite.java]: Do you think it's worth bringing Guava into this video quickstart (for ImmutableMap?)

Comment (ktalebian) [reply exampleVideoInvite/src/main/java/com/twilio/video/examples/videoinvite/notify/api/model/Invite.java]: @jefflinwood  sorry I was out on PTO.

Nope it is not worth it if you don't have that package already included. Ignore this comment then.

**Comment by zackm0571** [comment]
@jefflinwood @ktalebian I'm getting a 500 when using these changes. Log:

```
11:31:53.673 [Thread-1] INFO  s.e.jetty.EmbeddedJettyServer - == Spark has ignited ...
11:31:53.673 [Thread-1] INFO  s.e.jetty.EmbeddedJettyServer - >> Listening on 0.0.0.0:4567
11:31:53.675 [Thread-1] INFO  org.eclipse.jetty.server.Server - jetty-9.4.z-SNAPSHOT; built: 2018-08-30T13:59:14.071Z; git: 27208684755d94a92186989f695db2d7b21ebc51; jvm 1.8.0_192-b12
11:31:53.705 [Thread-1] INFO  org.eclipse.jetty.server.session - DefaultSessionIdManager workerName=node0
11:31:53.705 [Thread-1] INFO  org.eclipse.jetty.server.session - No SessionScavenger set, using defaults
11:31:53.706 [Thread-1] INFO  org.eclipse.jetty.server.session - node0 Scavenging every 660000ms
11:31:53.761 [Thread-1] INFO  o.e.jetty.server.AbstractConnector - Started ServerConnector@63bb65c1{HTTP/1.1,[http/1.1]}{0.0.0.0:4567}
11:31:53.764 [Thread-1] INFO  org.eclipse.jetty.server.Server - Started @476ms
11:32:00.139 [qtp1023454306-17] INFO  com.twilio.LoggingFilter -
Request GET /token HTTP/1.1 HEADERS:[Accept-Encoding:gzip, Host:eeb2e318.ngrok.io, User-Agent:okhttp/3.6.0, X-Forwarded-For:209.194.170.247, X-Forwarded-Proto:https]
Response 200 HEADERS:[Date:Thu, 07 Mar 2019 19:31:59 GMT, Content-Type:application/json]
11:32:00.918 [qtp1023454306-13] ERROR com.twilio.ServerApp - Exception creating binding: The requested resource /Services/null/Bindings was not found
com.twilio.exception.ApiException: The requested resource /Services/null/Bindings was not found
        at com.twilio.rest.notify.v1.service.BindingCreator.create(BindingCreator.java:152)
        at com.twilio.rest.notify.v1.service.BindingCreator.create(BindingCreator.java:27)
        at com.twilio.base.Creator.create(Creator.java:45)
        at com.twilio.ServerApp.lambda$main$6(ServerApp.java:150)
        at spark.RouteImpl$1.handle(RouteImpl.java:72)
        at spark.http.matching.Routes.execute(Routes.java:61)
        at spark.http.matching.MatcherFilter.doFilter(MatcherFilter.java:134)
        at spark.embeddedserver.jetty.JettyHandler.doHandle(JettyHandler.java:50)
        at org.eclipse.jetty.server.session.SessionHandler.doScope(SessionHandler.java:1568)
        at org.eclipse.jetty.server.handler.ScopedHandler.handle(ScopedHandler.java:144)
        at org.eclipse.jetty.server.handler.HandlerWrapper.handle(HandlerWrapper.java:132)
        at org.eclipse.jetty.server.Server.handle(Server.java:503)
        at org.eclipse.jetty.server.HttpChannel.handle(HttpChannel.java:364)
        at org.eclipse.jetty.server.HttpConnection.onFillable(HttpConnection.java:260)
        at org.eclipse.jetty.io.AbstractConnection$ReadCallback.succeeded(AbstractConnection.java:305)
        at org.eclipse.jetty.io.FillInterest.fillable(FillInterest.java:103)
        at org.eclipse.jetty.io.ChannelEndPoint$2.run(ChannelEndPoint.java:118)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.runTask(EatWhatYouKill.java:333)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.doProduce(EatWhatYouKill.java:310)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.tryProduce(EatWhatYouKill.java:168)
        at org.eclipse.jetty.util.thread.strategy.EatWhatYouKill.produce(EatWhatYouKill.java:132)
        at org.eclipse.jetty.util.thread.QueuedThreadPool.runJob(QueuedThreadPool.java:765)
        at org.eclipse.jetty.util.thread.QueuedThreadPool$2.run(QueuedThreadPool.java:683)
        at java.lang.Thread.run(Thread.java:748)
11:32:00.919 [qtp1023454306-13] INFO  com.twilio.LoggingFilter -
Request POST /register HTTP/1.1 HEADERS:[Accept-Encoding:gzip, Content-Length:290, Content-Type:application/json; charset=UTF-8, Host:eeb2e318.ngrok.io, User-Agent:okhttp/3.6.0, X-Forwarded-For:209.194.170.247, X-Forwarded-Proto:https]
Response 500 HEADERS:[Date:Thu, 07 Mar 2019 19:32:00 GMT, Content-Type:application/json]
```
Can you let me know if you can reproduce?

**Comment by jefflinwood** [comment]
@zackm0571 - looking at the error, it looks like the Service SID isn't found:

/Services/null/Bindings

Does that help narrow this down?

**Comment by zackm0571** [comment]
Tested locally and it worked as expected.

**Pull request merged**